src/python/espressomd/interactions.pyx

```diff
@@ -1696,7 +1686,7 @@ class BondedInteraction(ScriptInterfaceHelper):
                 params.update(kwargs)
                 self.validate_params(params)
                 super().__init__(*args, **params)
-                self._check_keys(params.keys(), check_required=True)
+                utils.check_valid_keys(self.valid_keys(), kwargs.keys())
```

Comment:
Note: it was not necessary to check required keys here because `super().__init__(*args, **params)` already raises a `RuntimeError` if a required key is missing.